### ISSUE TITLE
Addition of 'Unread messages' link to My Cyclescape

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#824]
A feature request:

Would be useful to have a link on the My Cyclescape listing, for filtering to unread messages only.

This would need to be implemented server-side, not in Javascript, as there is pagination (i.e. there's not much point filtering on the page of an already paginated list).

I suggest this simply be a link floated to the right, in line with  "Latest updates on the discussion threads you are subscribed to" which says: "Unread messages only" and when clicked on shows unread only, but the link then changes to "All messages".

**Comment by nikolai-b** [comment]
Similar to #773 but that is asking for users own posts but the principal is that My Cyclescape is not showing users quite what they want.  Instead of adding more links and features I think we might want to streamline the page, it already has a lot on it.  Instead of an additional link to unread items we could sort the list unread items first?

**Comment by mvl22** [comment]
> Instead of an additional link to unread items we could sort the list unread items first?

Definitely don't do that - that will put them out of order.

There is the ongoing need for a wider redesign but I think on balance the addition of an 'Unread messages' link is not too much clutter for now, and quite normal for this kind of listing.

**Comment by roxannedebeaux** [comment]
Yes, please don't make 'unread first' the default, that would be very
problematic. It's not an urgent requirement (I would probably prefer easier
use from mobile devices above everything else, but I know that it not easy)

I think sorting for unread is something that will help members to engage
with issues that have slipped past the first page. We have so much going on
now that it is easy to miss things.

Kind Regards,

Roxanne De Beaux
Executive Director

Usual working days MON - THURS

*CAMCYCLE**p: *07482 735174    *e: *roxanne.debeaux@camcycle.org.uk    *w: *
camcycle.org.uk <http://www.camcycle.org.uk/>*    t: *@camcycle
<http://www.twitter.com/camcycle>
*Join:*camcycle.org.uk/membership
<http://www.camcycle.org.uk/membership/join>*     Donate: *Text "CYCC11 £5"
to 70070
*Charity no. 1138098*
